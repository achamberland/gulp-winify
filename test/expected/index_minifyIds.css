/*Class */

.¢ {
  color: #0f0;
}

/* Tag with class */

li.¢ {
  border: 1px solid #66f;
}

/* Class with tag */

.£ a {
  text-decoration: none;
}

/* Tag with child class */

header .¢ {
  background: #ff6;
}

/* Multi-selector rules */

.¢ a,
.¢ p,
.¢ h1 {
  font-weight: 700;
}

/* :not pseudo-selectors with class */

.¢:not(.¤) {
  opacity: 1;
}

/* :not pseudo-selectors with another pseudo-selector */

header li.¢:not(:hover) {
  color: #f00;
}

/* Pseudo-selector with a :not pseudo-selector & class */

header li:hover:not(.¢) {
  color: #ff0;
}

/* To add and support: attr[class='...'] and all other alt equality signs */

#¢ {
  visibility: hidden;
}

div {
  background: 'hot pink';
}

header ul li {
  list-style: none;
}