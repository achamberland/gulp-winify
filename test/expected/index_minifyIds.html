<!--<![CDATA[--><!doctype html>
<!--]]-->
<html>

<head>

  <!-- External styles -->
  <link rel="stylesheet" src="build/demo.css">

  <!-- External scripts -->
  <link rel="stylesheet" src="build/demo.css">


  <!-- Embedded styles -->
  <style>/* Tag styles */

div {
  border: 1px solid #000;
}

nav li {
  display: inline-block;
}

/* Class styles */

.¢ {
  background: #000;
}

/* Id styles */

#¢ {
  float: right;
  padding: 20px;
  background: "hot pink";
}

/* Combo styles */

nav .£ {
  color: #f66;
}

article #£ {
  color: #66f;
}

</style>

  <!-- Embedded scripts -->
  <script type="text/javascript">
    
    var d = document.registerElement('d', {
      prototype: Object.create(HTMLDivElement.prototype)
    });

  </script>

</head>

<body>

  <div class="container ¢">


  <header>
    <nav>
      <ul>
        <li class="active £">
          <a href="#" id="Home ¤">Home</a>
        </li>
        <li>
          <a href="#" id="About ¥">About</a>
        </li>
      </ul>
    </nav>
  </header>

  <article>
    <div id="page-top ¦">Top of article</div>

    <h1></h1>

    <div class="shady § another-class ¨">
      Shaded text
    </div>

    <span class="bold ©" style="display:none"> </span>



    <tag1 class="class1 ª"></tag1>
    <tag11 class="class11 «"></tag11>
    <tag12>
    <tag13>
    <tag14>
    <tag15>
    <tag16>
    <tag17>
    <tag189>
    <tag19>
    <tag10>
    <tag112>
    <tag1123>
    <tag12312>
    <tag13213>
    <tag132113>
    <tag1323222>
    <tag1333>
    <tag33>
    <tag434>
    <tag1></tag1>
    <tag5352>
    <tag6546>
    <tag4562>
    <tag654>
    <tag77>
    <tag745>
    <tag2432>
    <tag754>
    <tag352>
    <tag2810>
    <tag3432>
    <tag1111111>
    <tag13231>
    <tag23123>
    <tag73567>
    <tag7468678>
    <tag946784>
    <tag345>
    <tag133456354>
    <tag1124>
    <tag12441>
    <tag14636457>
    <tag74251>
    <tag754271>
    <tag445271>
    <tag5471>
    <tag24574571>
    <tag5741>
    <tag545471>
    <tag4251>
    <tag4571>
    <tag45745271>
    <tag5471>
    <tag27241>
    <tag541>
    <tag75471>
    <tag5651>
    <tag64561>
    <tag5631>
    <tag56561>
    <tag56154>
    <tag561>
    <tag661>
    <tag6666561>

      <!-- Unclosed/misspelled tag -->
      <div>

    </div></tag6666561></tag661></tag561></tag56154></tag56561></tag5631></tag64561></tag5651></tag75471></tag541></tag27241></tag5471></tag45745271></tag4571></tag4251></tag545471></tag5741></tag24574571></tag5471></tag445271></tag754271></tag74251></tag14636457></tag12441></tag1124></tag133456354></tag345></tag946784></tag7468678></tag73567></tag23123></tag13231></tag1111111></tag3432></tag2810></tag352></tag754></tag2432></tag745></tag77></tag654></tag4562></tag6546></tag5352></tag434></tag33></tag1333></tag1323222></tag132113></tag13213></tag12312></tag1123></tag112></tag10></tag19></tag189></tag17></tag16></tag15></tag14></tag13></tag12></article>

  </div>

</body>


</html>